### ISSUE TITLE
Lastest Whatsapp Update no more reply!

### ISSUE THREAD

**Issue by ziadkiwan** [mohak1712/Insta-Chat#9]
Dear,

After the latestest whatsapp update 2.18.65 replying through the application is not working anymore! any idea how to solve it ! thank you!

**Comment by mohak1712** [comment]
After updating to the latest version even I have this issue
I will check this
Seems like some action has changed

**Comment by ziadkiwan** [comment]
Yep i was inspecting the code through debugging the wearable.getactions is returning null now :/

**Comment by ashutosh1127** [comment]
HI ,
Any solution to above .

Thanks

**Comment by mohak1712** [comment]
@ashutosh1127 as of now not updating the app to the latest version is the only solution. However I will look into it asap

**Comment by arya-ayush** [comment]
Please tell the concept that you have applied to send message to whatsapp contact without opening whatsapp.

**Comment by arya-ayush** [comment]
Without knowing the concept, it is difficult to figure out the issue

**Comment by mohak1712** [comment]
@arya-ayush For reply feature i use actions present in notification. These actions allow wearable devices to send reply directly.
However in the latest version the action is null so we are unable to send the message.

**Comment by arya-ayush** [comment]
Thanks. I will see
